.gitignore

```diff
@@ -1,6 +1,7 @@
 .hasura-dev-python-venv
 server/tests-py/.devsh_version
 npm-debug.log
+dist-newstyle/
```

Comment:
Why was this not done earlier?

Reply:
I am not sure about this though, this might be because of my dev setup, but I don't see this as a significant issue. :sweat_smile:

Reply:
Yeah, this is definitely not an issue. I was just wondering why `dist-newstyle` was not in `.gitignore`. Anyway, I am marking this as resolved.